lib/auth0/api/v2/users.rb

```diff
@@ -185,12 +185,137 @@ def user_logs(user_id, options = {})
+        #
+        # @return [json] Returns roles for the given user_id.
+        def get_roles(user_id, options = {})
+          raise Auth0::MissingUserId, 'Must supply a valid user_id' if user_id.to_s.empty?
+          path = "#{users_path}/#{user_id}/roles"
+          request_params = {
+            per_page:       options.fetch(:per_page, nil),
+            page:           options.fetch(:page, nil),
+            include_totals: options.fetch(:include_totals, nil)
+          }
+          get(path, request_params)
+        end
+
+        # Remove one or more roles from a specific user.
+        # @see https://auth0.com/docs/api/management/v2#!/Users/delete_user_roles
+        #
+        # @param user_id [string] The user_id of the roles to remove.
+        # @param roles [array] An array of role names to remove.
+        def remove_roles(user_id, roles)
+          raise Auth0::MissingUserId, 'Must supply a valid user_id' if user_id.to_s.empty?
+          validate_roles_array roles
+          path = "#{users_path}/#{user_id}/roles"
+          delete(path, { roles: roles })
+        end
+
+        # Add one or more roles to a specific user.
+        # @see https://auth0.com/docs/api/management/v2#!/Users/post_user_roles
+        #
+        # @param user_id [string] The user_id of the roles to add.
+        # @param roles [array] An array of role names to add.
+        def add_roles(user_id, roles)
+          raise Auth0::MissingUserId, 'Must supply a valid user_id' if user_id.to_s.empty?
+          validate_roles_array roles
+          path = "#{users_path}/#{user_id}/roles"
+          post(path, { roles: roles })
+        end
+
+        # Get all Guardian enrollments for a specific user
+        # @see https://auth0.com/docs/api/management/v2#!/Users/get_enrollments
+        #
+        # @param user_id [string] The user_id of the enrollments to get.
+        #
+        # @return [json] Returns Guardian enrollments for the given user_id.
+        def get_enrollments(user_id)
+          raise Auth0::MissingUserId, 'Must supply a valid user_id' if user_id.to_s.empty?
+          get "#{users_path}/#{user_id}/enrollments"
+        end
+
+        # Get all permissions for a specific user.
+        # @see https://auth0.com/docs/api/management/v2#!/Users/get_permissions
+        #
+        # @param user_id [string] The user_id of the permissions to get.
+        #
+        # @return [json] Returns permissions for the given user_id.
+        def get_permissions(user_id)
+          raise Auth0::MissingUserId, 'Must supply a valid user_id' if user_id.to_s.empty?
+          get "#{users_path}/#{user_id}/permissions"
+        end
+
+        # Remove one or more permissions from a specific user.
+        # @see https://auth0.com/docs/api/management/v2#!/Users/delete_permissions
+        #
+        # @param user_id [string] The user_id of the permissions to remove.
+        # @param permissions [array] An array of Permissions to remove.
```

Comment:
Random uppercase P on permissions.

Reply:
"Permissions" here refers to the struct I made for this PR.